src/components/WorkOrder/Photos/hooks/uploadFiles/fileUploadStatusLogger.ts

```diff
@@ -0,0 +1,19 @@
+const fileUploadStatusLogger = (
+  totalFiles: number,
+  logCallback: (message: string) => void
+) => {
```

Comment:
The type inference for this function's return type is `void` when it should be some callable / `() => void`